docs/dm/attestation-results/attestation-results.cddl

```diff
@@ -0,0 +1,43 @@
+start = attestation-results
+
+$ar-status /= ar.status-failure
+$ar-status /= ar.status-success
+$ar-status /= ar.status-unknown
+
+attestation-results = {
+  ar.status => $ar-status
+  ar.trust-vector => ar-trust-vector
+  ? ar.raw-evidence => ar-bytes
```

Comment:
Is the intent here just to be bytes or a structured replay of evidence (potentially with 'checked' flags)? RP may well have additional appraisal path for fields which require RV or Endorsement checking beyond scope of verifier. e.g. software workload measurements on a verified platform

Reply:
We debated this quite extensively, and the rationale for keeping it a verbatim copy rather than a re-structured thing is that this way we avoid adding extra interface surface on a per-token-format basis, and therefore avoid any coupling that is not strictly required.  Note that, since the format is extensible, this doesn't preclude a closer RP-Verifier relationship where the Verifier takes care of making Evidence into something easier to consume by the RP.

Reply:
I can see the argument both ways, but was inspired by the spirit of veraison taking care of all token specific details and hence being in an ideal position to produce a highly readable / processable form of the evidence for an RP (it's available as an internal structure after all)! Maybe there should be an optional ar.found-evidence or similar placeholder to support further discussion.

Reply:
If we want to add it I'd make it a "Veraison feature" using a (TBD) extension point rather than a core feature that all verifiers implementing this interface need to provide, i.e.:
```
; to be added to the attestation-results map
$$attestation-results-extension //= (
  ar-veraison.found-evidence => ar-veraison-found-evidence
)

ar-veraison-found-evidence = {
  $ar-label => any
}
```

Reply:
Understood - I had missed the nuance that the CDDL work was for a standard implementation rather than just veraison

Reply:
cool, I'm adding the (currently) missing extension point then!

Reply:
Done in 9717e6a